waspc/test/JsImportTest.hs

```diff
@@ -0,0 +1,53 @@
+module JsImportTest where
+
+import StrongPath (Dir', File', Path, Posix, Rel, (</>))
+import qualified StrongPath as SP
+import Test.Tasty.Hspec (Spec, describe, it, shouldBe)
+import Wasp.JsImport
+
+spec_JsImportTest :: Spec
+spec_JsImportTest = do
+  describe "makeJsImport" $ do
+    it "makes JsImport with default import from a path" $ do
+      makeJsImport pathToJsFile (JsImportModule "test")
+        `shouldBe` JsImport
+          { _path = pathToJsFile,
+            _name = JsImportModule "test",
+            _importAlias = Nothing
+          }
+  describe "applyJsImportAlias" $ do
+    it "applies alias to JsImport" $ do
+      applyJsImportAlias
+        (Just "alias")
+        (makeJsImport pathToJsFile (JsImportModule "test"))
+        `shouldBe` JsImport
+          { _path = pathToJsFile,
+            _name = JsImportModule "test",
+            _importAlias = Just "alias"
+          }
+  describe "generateJsImportStatement" $ do
+    it "generates import statement for default import" $ do
+      getJsImportStmtAndIdentifier
+        (makeJsImport pathToJsFile (JsImportModule "test"))
+        `shouldBe` ("import test from './ext-src/folder/test.js'", "test")
```

Comment:
`./ext-src/folder/test.js` here is repeated in many places below. It would be good to remove that duplication.
Also, it is connected with `pathToJsFile`, but that connection is not clear.